### PR TITLE
[skip ci] Added info about setting static IP addresses

### PR DESCRIPTION
**Pull request by stuclem**
Along with some other minor updates.

@andrewtchin, can you please look at the changes in doc/user_doc/vic_installation/vch_installer_examples.md and doc/user_doc/vic_installation/vch_installer_options.md? 

Thanks!

### CODE REVIEW

**Comment by stuclem** [comment]
Fixes #1470

File: doc/user_doc/vic_installation/vch_installer_examples.md [hunk 1]
```diff
@@ -110,6 +111,37 @@ This example deploys a virtual container host with the following configuration:
...
+## Deploy a Virtual Container Host with a Static IP Address on the Different Networks ##
+
+If you specify networks for any or all of the external, management, and client networks, you can deploy the virtual container host so that it has a static IP address on those networks. 
+
+This example deploys a virtual container host with the following configuration:
+
+- Specifies the user name, password, datacenter, cluster, image store, bridge network, and name for the virtual container host.
+- Directs external, management, and Docker API traffic to network 1, network 2, and network 3 respectively. Note that the network names are wrapped in quotes, because they contain spaces. Use single quotes if you are using `vic-machine` on a Linux or Mac OS system and double quotes on a Windows system.
+- Identifies a DNS server for static IP addresses.
+- Obtains a static IP address for the virtual container host on each of the external, management, and client networks.
```
Comment (andrewtchin) [inline doc/user_doc/vic_installation/vch_installer_examples.md]: Sets a static IP address

Comment (stuclem) [inline doc/user_doc/vic_installation/vch_installer_examples.md]: Fixed

File: doc/user_doc/vic_installation/vch_installer_options.md [hunk 2]
```diff
@@ -129,6 +133,18 @@ When you specify the bridge network IP range, you specify the IP range as a CIDR
 
 <pre>--bridge-network-range 192.168.100.0/24</pre>
 
+### `dns-server` ###
+
+Short name: None
+
+A DNS server to use if you specify static IP addresses for the virtual container host on the client, external, and management networks. You can specify `dns-server` multiple times, to configure multiple DNS servers.  
+
+If you specify `dns-server` but you do not specify a static IP addresses for one or more of the client, external, and management networks, `vic-machine create` ignores the `dns-server` setting for that network and uses the DNS servers that are provided by DHCP. You can therefore use a mixture of static and DHCP addresses for the virtual container host on the different networks.
```
Comment (andrewtchin) [inline doc/user_doc/vic_installation/vch_installer_options.md]: If you specify `dns-server` but you do not specify a static IP addresses for one or more of the client, external, and management networks, `vic-machine create` ignores the `dns-server` setting and uses the DNS servers that are provided by DHCP. 

If you use a mixture of static and DHCP addresses for the virtual container host on the different networks, the virtual container host will use the DNS servers specified by `dns-server` and those provided by DHCP.

**Comment by andrewtchin** [comment]
lgtm

**Comment by stuclem** [comment]
Integrated final comment from @andrewtchin (thanks!) so LGTM too.

**Comment by stuclem** [comment]
:recycle:

**Pull request merged**